client/faucet/src/mint.rs

```diff
@@ -49,7 +50,23 @@ impl MintParams {
                 0, // sliding nonce
                 self.receiver(),
                 self.auth_key.prefix().to_vec(),
-                format!("No. {}", seq).as_bytes().to_vec(),
+                format!("No. {} VASP", seq).as_bytes().to_vec(),
+                false, /* add all currencies */
+            ),
+        )
+    }
+
+    fn create_designated_dealer_account_script(
+        &self,
+        seq: u64,
+    ) -> libra_types::transaction::TransactionPayload {
+        libra_types::transaction::TransactionPayload::Script(
+            transaction_builder_generated::stdlib::encode_create_designated_dealer_script(
+                self.currency_code(),
+                0, // sliding nonce
```

Comment:
shouldn't this be seq?

Reply:
encode_create_designated_dealer_script,

this is not creating transaction, it is script parameter.

Reply:
oh interesting, I had been inserting the sequence number here too in all my cases but it looks like that's not required